### PR TITLE
Add getMeta gRPC call and meta_table in py client

### LINKED ISSUE #2243: Support `getMeta` in client APIs
The client APIs do not currently support `getMeta`.  This is a very useful method when developing code.  To support the client APIs, `getMeta` must first be implemented in the gRPC layer.

### PR DESCRIPTION
**Pull request by jmao-denver**
Fixes #2243 #3620 

related to #3648 

TODO: Add getMeta in the  Go (@chipkent ) and CPP (@kosak ) clients

### CODE REVIEW

File: py/client/pydeephaven/table.py [hunk 1]
```diff
@@ -72,6 +73,12 @@ def _parse_schema(self, schema_header):
         reader = pa.ipc.open_stream(schema_header)
         self.schema = reader.schema
 
+    @property
+    def meta_table(self) -> Table:
```
Comment (devinrsmith) [inline py/client/pydeephaven/table.py:77]: Does this make sense as a `property`?

File: authorization/src/main/java/io/deephaven/auth/codegen/impl/TableServiceContextualAuthWiring.java
[the remark this answers is not in the document]
Comment (jmao-denver) [reply authorization/src/main/java/io/deephaven/auth/codegen/impl/TableServiceContextualAuthWiring.java:1]: Yeah, I had the same question and asked @niloc132 about it. @nbauernfeind your thoughts?

[the remark this answers is not in the document]
Comment (devinrsmith) [reply authorization/src/main/java/io/deephaven/auth/codegen/impl/TableServiceContextualAuthWiring.java:1]: The scripts are in `authorization-codegen/`. `authorization/README.md` w/ details.

File: py/client/pydeephaven/table.py (hunk 1 above)
Comment (jmao-denver) [reply py/client/pydeephaven/table.py:77]: I think you can argue either way. The server API also has it as a property.

Comment (devinrsmith) [reply py/client/pydeephaven/table.py:77]: I'm not fully aware of the pythonic semantics one might assume when something is marked as a property. That said, naively I would be surprised as an end-user if I call a property method and it ends up needing to do a round-trip RPC.

Comment (jmao-denver) [reply py/client/pydeephaven/table.py:77]: @property is the Pythonic way to implement a getter method, though not recommended for getters that take long time to finish.

**Review by niloc132 — DISMISSED** [review]

File: py/client/pydeephaven/table.py (hunk 1 above)
Comment (devinrsmith) [reply py/client/pydeephaven/table.py:78]: Isn't this a prime example of a method that might take a long time to finish?

**Review by devinrsmith — CHANGES_REQUESTED** [review]
The authorization wiring is incorrect, it needs to be done with the code generation. Right now the method names are incorrect.

File: py/client/pydeephaven/table.py (hunk 1 above)
Comment (jmao-denver) [reply py/client/pydeephaven/table.py:78]: yes, and I guess it also depends on the real use case of it. OTOH we also want to keep the client API/server API consistent.

File: server/src/main/java/io/deephaven/server/table/ops/MetaTableGrpcImpl.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import io.deephaven.base.verify.Assert;
+import io.deephaven.engine.table.Table;
+import io.deephaven.proto.backplane.grpc.BatchTableRequest;
+import io.deephaven.proto.backplane.grpc.MetaTableRequest;
+import io.deephaven.server.session.SessionState;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class MetaTableGrpcImpl extends GrpcTableOperation<MetaTableRequest> {
```
Comment (chipkent) [inline server/src/main/java/io/deephaven/server/table/ops/MetaTableGrpcImpl.java:16]: Does this java stuff need docs?

**Review by devinrsmith — APPROVED** [review]

**Pull request merged**